src/main/java/org/grizz/factory/PlanetFactory.java

```diff
@@ -0,0 +1,28 @@
+package org.grizz.factory;
+
+import com.google.common.collect.Lists;
+import org.grizz.model.Building;
+import org.grizz.model.Planet;
+import org.grizz.model.User;
+import org.grizz.model.enummerations.BuildingType;
+import org.springframework.stereotype.Component;
+
+import java.util.Set;
+import java.util.stream.Collectors;
+
+@Component
+public class PlanetFactory {
+
```

Comment:
Enter enter enter. What for?! :D I've notice that you leave such things quite often. Please set up your auto-formatting to remove double enters :)